src/main/java/storagesystem/model/ReservationHandler.java

```diff
@@ -114,12 +116,8 @@ private boolean isObjectReservedBetween(IReservable object, Interval interval) {
      * @param object   The object which is to be reserved.
      */
     public void createReservation(IBorrower borrower, Interval interval, IReservable object) {
-        if (!isObjectReservedBetween(object, interval)) {
-            Reservation reservation = new Reservation(borrower, interval, object, ReservationStatus.PENDING);
-            reservations.add(reservation);
-
-
-        }
+        Reservation reservation = new Reservation(borrower, interval, object, ReservationStatus.PENDING);
```

Comment:
No reason to create local variable.
```suggestion
       reservations.add(new Reservation(borrower, interval, object, ReservationStatus.PENDING));
```